enterprise/api/v1/views.py

```diff
@@ -229,7 +229,7 @@ def retrieve(self, request, pk=None):  # pylint: disable=invalid-name
             request (HttpRequest): Current request
             pk (int): Course catalog identifier
 
-        Returns:
+        Returns
```

Comment:
@snowcrshd Inconsistentsy: in this file (and some others) "Returns" are not underlined, in some other files it is. I.e. here it is just

    Returns

while in those other files it is

    Returns
    -----------